src/gui/HelpWidget.cpp

```diff
@@ -10,15 +10,15 @@
 #include "HelpWidget.h"
 #include "Globals.h"
 
-HelpWidget::HelpWidget() : QVBoxWidget() {
+HelpWidget::HelpWidget(QWidget* parent) : QVBoxWidget() {
```

Comment:
Should QVBoxWidget() get *parent* as parameter? By the book, yes. The widget's parent will however be replaced when added to the document area.